packages/components/organisms/f-takeawaypay-activation/src/components/ActivationNotLoggedIn.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div>
+        <bag-celebrate-bg-icon :class="$style['c-takeawaypayActivation-icon']" />
+
+        <p :class="$style['c-takeawaypayActivation-title']">
+            {{ $t('messages.titleActivationNotLoggedIn') }}
+        </p>
+
+        <p>
+            {{ $t('messages.descriptionActivationNotLoggedIn') }}
+        </p>
+
+        <div :class="$style['c-takeawaypayActivation-actions']">
+            <f-button
+                :href="loginUrl"
+                data-test-id="takeawaypay-activation-not-loggedin-login-link"
+                button-type="primary"
+                button-size="large"
+                is-full-width>
+                {{ $t('actions.loginToAccount2') }}
+            </f-button>
+
+            <f-button
+                :href="registrationUrl"
+                data-test-id="takeawaypay-activation-not-loggedin-registration-link"
+                button-type="outline"
+                button-size="large"
+                is-full-width>
+                {{ $t('actions.createAccount') }}
+            </f-button>
+        </div>
+    </div>
+</template>
+
+<script>
+import FButton from '@justeat/f-button';
+import '@justeat/f-button/dist/f-button.css';
+import { BagCelebrateBgIcon } from '@justeat/f-vue-icons';
```

Comment:
This icon is different from the one on the ticket - I'm assuming because the correct one isn't currently available in the icon library - are you planning to add the new one to the icon library?

Reply:
Yep, exactly. Added existing icon as a placeholder and will need add other icons to the f-vue-icons

Reply:
f-vue-icons should be used only for unified icons like arrows, plusses, driver/cyclist icons etc.  Icons which are specific for a component should not be added to f-vue-icons. It is still an open discussion what is the best way to store icons like this.

Reply:
@dandel10n Thanks, good to know. Thought to add other "bag" icons there, as that one was already used in the f-registration component. But happy to use the agreed approach.